### ISSUE TITLE
19 Vulnerabilities

### ISSUE THREAD

**Issue by thesolomon-tech** [lxieyang/chrome-extension-boilerplate-react#218]
Hello,

When running npm install for the first time I encountered 19 vulnerabilities: 1 low, 7 moderate, 9 high, 2 critical.

Thanks

**Comment by Anonymous-Humanoid** [comment]
Yes, this package uses some outdated packages. Fortunately, npm publishes security fixes. Simply run `npm audit fix` and they should be resolved.
